packages/app/src/messages/errors/resend-message.error.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { MessagesError } from "App/messages/constants"
+
+export class ResendMessageError extends Error {
+  public type = MessagesError.ResendMessageError
```

Comment:
Really small thing, but this field might be marked as `readonly`.

Reply:
https://github.com/mudita/mudita-center/pull/802/commits/a2c395e2a556ca84e335df2d6fc42eea82db25ed